utils.js

```diff
@@ -0,0 +1,22 @@
+export const isElementAlreadyRegistered = (elName) => {
+  var registeredElement;
+  if (window.Polymer && Polymer.telemetry) {
+    registeredElement = Polymer.telemetry.registrations.filter((el) => el.is == elName);
+
+    if (registeredElement) {
+      console.warn(elName, "is already registered as a custom polymer element.");
+    }
+
+    return !!registeredElement;
```

Comment:
As per this condition, whenever site uses `Polymer`, `customElements` registry is never checked. so, it's a logical issue.